src/fsharp/FSharp.Compiler.LanguageServer/LspTypes.fs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.LanguageServer
+
+// Types as defined at https://microsoft.github.io/language-server-protocol/specification
+
+/// Represents a zero-based line and column of a text document.
+type Position = {
+    line: int
+    character: int
+}
+
+type Range = {
+    start: Position
+    ``end``: Position
```

Comment:
Upper case will make it so you won't need the backticks

Reply:
Same comment as above about `[<JsonProperty("name")>]`.